Modules/Modules/Exposing Paths with the pub Keyword/task.md

```diff
@@ -0,0 +1,71 @@
+### Exposing Paths with the pub Keyword
+
+Let’s return to the error in the previous example that told us the `hosting` module is private. We want the `eat_at_restaurant` function in the parent module to have access to the `add_to_waitlist` function in the child module, so we mark the `hosting` module with the `pub` keyword, as shown in the listing below.
+
+```rust
+    mod front_of_house {
+        pub mod hosting {
+            fn add_to_waitlist() {}
+        }
+    }
+
+    pub fn eat_at_restaurant() {
+        // Absolute path
+        crate::front_of_house::hosting::add_to_waitlist();
+
+        // Relative path
+        front_of_house::hosting::add_to_waitlist();
+    }
+```
+
+##### Declaring the hosting module as pub touse it from eat_at_restaurant
+
+Unfortunately, the code from the snippet above still results in an error, as shown below.
```

Comment:
Unfortunately, the code from the snippet above still results in an error as shown below.